Makefile

```diff
@@ -36,25 +36,27 @@ deps: ## Installs dependencies
 	fi; \
 		poetry install --quiet || poetry install; \
 	fi; \
+	
+format: ## Formats your code automatically.
+	poetry run isort .
+	poetry run black .
 
-lint: deps ## Lints code 
+lint: ## Lints code 
 	poetry run ruff .
 
-run: deps ## Start the api locally on port 28900.
+run: ## Start the api locally on port 28900.
 	FLASK_APP=${FLASK_APP} poetry run flask run --port ${BERLIN_API_PORT}
 
-run-container: deps
-	docker run --env START_TIME='${START_TIME}' -e GIT_COMMIT="${GIT_COMMIT}" -e VERSION="${VERSION}" -ti berlin_api
+run-container:
+	docker run --env BUILD_TIME='${BUILD_TIME}' -e GIT_COMMIT="${GIT_COMMIT}" -e VERSION="${VERSION}" -ti berlin_api
 
-test: deps ## Runs all available tests and generates a coverage report located in htmlcov
-	poetry run ./scripts/run_tests_unit.sh
 
-test-component: deps ## Makes sure dep are installed and runs component tests
-	poetry run pytest tests/api
+test-component: ## runs component tests
+	poetry run pytest -v tests/api
 
-format: deps ## Formats your code automatically.
-	poetry run isort .
-	poetry run black .
+
+unit: ## runs component tests
+	poetry run pytest -v tests/unit
```

Comment:
Please can this be changed back to 'test' for consistency.

Reply:
also the comment is inaccurate.

Reply:
Yeah I'll fix that but I changed it to unit so that it's consistent with category api